Routes/Question/Correct.js

```diff
@@ -6,27 +6,46 @@ var Tags = require('../Validator.js').Tags;
 var router = Express.Router({caseSensitive: true});
 var async = require('async');
  
-router.baseURL = '/Correct';
+router.baseURL = 'Qsts/Correct';
 
-router.get('/:msgId', function(req, res) {
+router.get('/', function(req, res) {
    var vld = req.validator;
    var cnn = req.cnn;
-   var msgId = req.params.msgId;
 
    async.waterfall([
    function(cb) {
-      req.cnn.chkQry('select UNIX_TIMESTAMP(whenMade) * 1000 as whenMade,' +
-       ' email, content from Message join Person on Message.prsId =' +
-       ' Person.id where Message.id = ?', parseInt(msgId), cb);
+      // Get questions that the user has answered correctly
+      cnn.chkQry('select qst.id, qst.title, qst.ownerId, ctg.title as' +
+       ' categoryTitle, qst.answer from Question as qst, Category as ctg,' +
+       ' PersonQuestion as pq where pq.personId = ? and ctg.id =' +
+       ' qst.categoryId and qst.id = pq.questionId;', req.session.id, cb);
    },
-   function(msgs, fields, cb) {
-      if (vld.check(msgs.length, Tags.notFound, null, cb)) {
-      		res.json(msgs[0]);
-          cnn.release();
-      }
-   
+   function(answers) {
```

Comment:
doesn't the last function in waterfall also need to take in & call the callback or else it will hang?